tests/integration/devfile/cmd_devfile_test_test.go

```diff
@@ -44,15 +44,22 @@ var _ = Describe("odo devfile test command tests", func() {
 
 	Context("Should show proper errors", func() {
 
-		It("should show error if component is not pushed", func() {
+		// used ";" as consolidating symbol as this spec covers multiple scenerios
+		It("should show error if component is not pushed; should error out if a non-existent command or a command from wrong group is specified", func() {
```

Comment:
`It` clause can be acts as a `then` clause in ginkgo. Of Course you can add multiple statement by using any separator like ```;```, ```/``` whatever but it will be a misuse of delimiter in a ```It``` description I believe. 
Moreover `It` clause should be a simple string which should explain what is the spec stand for and TBH adding a ```;``` looks like we are implementing our programming skills in writing a simple string 🙂 . Instead we can add the multiple statement in a single like.
```
Should show error if component is not pushed or specified commands either does not exist or belongs to wrong group
```
Can’t we ?

Reply:
consolidation is signified with `;` and I discussed this with @dharmit . And I agree that `;` makes it more readable at this point.

Reply:
Make sense. However I have gone through the whole ginkgo doc and nowhere it has been mentioned that we can use delimiter like `;` for separating multiple scenario in a single spec description. Also it is nowhere written we can not use this, afterall its a string message. As we are introducing `;` for the first time, I feel its better to document it in test guidelines https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc of our reference.

Reply:
@girishramnani As per https://github.com/openshift/odo/pull/3835#discussion_r478213783 could you please add `;` description in the test guideline.

Reply:
If you want i can add this change in a separate pr in the test guideline.